src/fields/NullableField.ts

```diff
@@ -22,13 +22,9 @@ export class NullableField<T extends FieldModel<unknown>> extends FieldModel<Nul
      * @returns {boolean}
      */
     override validate(value: unknown): true | never {
-        if (value) {
-            return this.model.validate(value);
-        }
-
-        if (![null, undefined].includes(value)) {
-            throw new TypeError(`value '${value}' is not 'null' or 'undefined'`);
-        }
+        if (value) return this.model.validate(value);
+        if (value !== null) throw new TypeError(`value '${value}' is not 'null' or 'undefined'`);
```

Comment:
since this is a `nullable` field, we should use `value != null` to check if the given value is `null` or `undefined`

```diff
- if (value !== null) throw new TypeError(`value '${value}' is not 'null' or 'undefined'`);
+ if (value != null) throw new TypeError(`value '${value}' is not 'null' or 'undefined'`);
```

you may need to suppress eslint error here

Reply:
It is because of eslint check.